webapp/channels/src/components/team_controller/actions/index.ts

```diff
@@ -50,8 +51,20 @@ export function initializeTeam(team: Team): ActionFunc<Team, ServerError> {
         if (license &&
             license.IsLicensed === 'true' &&
             (license.LDAPGroups === 'true' || customGroupEnabled)) {
+            const groupsParams: GetGroupsParams = {
+                filter_allow_reference: false,
+                page: 0,
+                per_page: 60,
+                include_member_count: true,
+                include_archived: false,
```

Comment:
@sbishel heads up, I changed `include_archived` to `false` here instead of `true`. On second thought there was no need to for the team controller to fetch archived groups